x/params/keeper/keeper_test.go

```diff
@@ -1,22 +1,20 @@
 package keeper_test
 
 import (
+	"github.com/cosmos/cosmos-sdk/testutil"
```

Comment:
nit: try to have consistent groupings. What editor do you use? It should be

(
 stdlib

everything else 

sdk
 )

Reply:
it's less about the IDE I think. It's the formatter, which by default is `goimports`. Is your IDE using `goimports`?